doc/ping.xml

```diff
@@ -204,6 +204,19 @@ xml:id="man.ping">
           kernel.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term>
+          <option>-e</option>
+        </term>
+        <listitem>
+          <para>Set identification field of the ECHO_REQUEST.
+          This implies SOCK_RAW option since setting identification
+          on SOCK_DGRAM doesn't work right now. The value of
```

Comment:
BTW looking at kernel sources, do you know how is `iph->id` (the identifier) set for ICMP datagram socket (it's not touched in `net/ipv[46]/icmp.c`).